### ISSUE TITLE
properties undefined error

### ISSUE THREAD

**Issue by jillesvangurp** [errata-ai/vale-vscode#98]
I can't get vale working with vs-code. Current version vs-code installs is still 0.15.0 instead of the new release.

Vs-code seems to not pick up the new 0.15.1 release of the extension yet (maybe that fixes this error?). I'll close this issue later if it updates to this and that fixes the issue. Is there any way to force this or speed it up?

Meanwhile, here's my setup.

I've set the configuration path and installed vale via homebrew (v2.20.1). It seems to work from the command line with the alias I get via my dotfiles (`alias vale=vale --config ~/.vale.ini`)

```bash
vale ./articles/article.md
```

I've configured vs-code as follows

- binary is in /opt/homebrew/bin/vale
- config is in /Users/jillesvangurp/.vale.ini

When I check the dev console, I see the following error.

```
Cannot read properties of undefined (reading 'message'): TypeError: Cannot read properties of undefined (reading 'message')
	at d.<anonymous> (/Users/jillesvangurp/.vscode/extensions/errata-ai.vale-server-0.15.0/dist/extension.js:1:4499)
	at Generator.next (<anonymous>)
	at /Users/jillesvangurp/.vscode/extensions/errata-ai.vale-server-0.15.0/dist/extension.js:1:2770
	at new Promise (<anonymous>)
	at s (/Users/jillesvangurp/.vscode/extensions/errata-ai.vale-server-0.15.0/dist/extension.js:1:2515)
	at d.provideCodeActions (/Users/jillesvangurp/.vscode/extensions/errata-ai.vale-server-0.15.0/dist/extension.js:1:4430)
	at A.provideCodeActions (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:87:42854)
	at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:87:73099
	at Z._withAdapter (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:87:67207)
	at Z.$provideCodeActions (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:87:73075)
	at o._doInvokeHandler (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:95:13691)
	at o._invokeHandler (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:95:13373)
	at o._receiveRequest (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:95:12092)
	at o._receiveOneMessage (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:95:10845)
	at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:95:8952
	at w.invoke (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:61:145)
	at b.deliver (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:61:2266)
	at p.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:61:1844)
	at c.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:69:19006)
	at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:17079
	at w.invoke (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:61:145)
	at b.deliver (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:61:2266)
	at p.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:61:1844)
	at c.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:69:19006)
	at i._receiveMessage (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:69:23593)
	at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:69:21127
	at w.invoke (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:61:145)
	at b.deliver (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:61:2266)
	at p.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:61:1844)
	at p.acceptChunk (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:69:15837)
	at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:69:14967
	at Socket.l (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:69:26303)
	at Socket.emit (node:events:390:28)
	at addChunk (node:internal/streams/readable:315:12)
	at readableAddChunk (node:internal/streams/readable:289:9)
	at Socket.Readable.push (node:internal/streams/readable:228:10)
	at Pipe.onStreamRead (node:internal/stream_base_commons:199:23)
```

**Comment by ChrisChinchilla** [comment]
@jillesvangurp Hey! Sorry, it's not actually released, only @jdkato seems to have the access code to make new releases to the VSCode marketplace, despite our best efforts to expand that… So will have to wait for him to come online later today. Sorry! Did test the last release and get no errors, hopefully this fix release does solve everything.

**Comment by jillesvangurp** [comment]
OK, will test when it is out.

**Comment by jessicajaniuk** [comment]
This is the same issue as #93. I attempted to upgrade to `0.15.1` today and the same issues are present.

**Comment by jdkato** [comment]
This should be fixed in v0.15.2 (released today).
